src/frontends/onnx/frontend/src/op/quant_conv.cpp

```diff
@@ -2,22 +2,25 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-// Disabled in CMakeList
-// Update to higher opset required
+// Disabled in CMakeLists.txt
+// Update to higher opset is required
 
-#include <cstddef>
-#include <memory>
-#include <vector>
+#if 0
```

Comment:
Why `if 0`?

Reply:
This file is not used, previously it was skipped in cmake